rolling-shutter/keyperimpl/gnosis/metrics.go

```diff
@@ -0,0 +1,95 @@
+package gnosis
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var metricsTxPointer = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "gnosis",
+		Name:      "tx_pointer",
+		Help:      "Current value of the tx pointer",
+	},
+	[]string{"eon"},
+)
+
+var metricsTxPointerAge = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "gnosis",
+		Name:      "tx_pointer_age",
+		Help:      "Current age of the tx pointer",
+	},
+	[]string{"eon"},
+)
+
+var metricsLatestTxSubmittedEventIndex = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "gnosis",
+		Name:      "latest_tx_submitted_event_index",
+		Help:      "Index of the latest TxSubmitted event",
+	},
+	[]string{"eon"},
+)
+
+var metricsTxSubmittedEventsSyncedUntil = prometheus.NewGauge(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "gnosis",
+		Name:      "tx_submitted_events_synced_until",
+		Help:      "Block number until which TxSubmitted events have been synced",
+	},
+)
+
+var metricsValidatorRegistrationsSyncedUntil = prometheus.NewGauge(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "gnosis",
+		Name:      "validator_registrations_synced_until",
+		Help:      "Block number until which validator registration events have been synced",
+	},
+)
+
+var metricsNumValidatorRegistrations = prometheus.NewGauge(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "gnosis",
+		Name:      "num_validator_registrations",
```

Comment:
Similar to above, according to the naming conventions it should be `validator_registrations_total`.